test/Deserializers/Tests/Phpunit/Exceptions/InvalidAttributeExceptionTest.php

```diff
@@ -3,40 +3,37 @@
 namespace Deserializers\Tests\Phpunit\Exceptions;
 
 use Deserializers\Exceptions\InvalidAttributeException;
+use Exception;
+use PHPUnit_Framework_TestCase;
 
 /**
  * @covers Deserializers\Exceptions\InvalidAttributeException
  *
  * @licence GNU GPL v2+
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ * @author Thiemo Mättig
  */
-class InvalidAttributeExceptionTest extends \PHPUnit_Framework_TestCase {
+class InvalidAttributeExceptionTest extends PHPUnit_Framework_TestCase {
 
 	public function testConstructorWithOnlyRequiredArguments() {
-		$attributeName = 'theGame';
-		$attributeValue = 'youJustLostIt';
```

Comment:
Because such strings do not add helpful knowledge to the code. They add confusion.